core/notifications/src/email_reminder_projection/mod.rs

```diff
@@ -13,7 +13,13 @@ pub struct EmailReminderProjection {
     _pool: PgPool,
 }
 
+const PAGINATION_BATCH_SIZE: i64 = 1000;
+
 impl EmailReminderProjection {
+    const ACCOUNT_LIVENESS_THRESHOLD_DAYS: i64 = 21;
+    const ACCOUNT_AGED_THRESHOLD_DAYS: i64 = 21;
+    const NOTIFICATION_COOL_OFF_THRESHOLD_DAYS: i64 = 90;
```

Comment:
why not directly convert to duration here ?

Reply:
Agree